### PR TITLE
[EGD-7980] New script for get bt from rel build

### PR DESCRIPTION
**Pull request by lucckb**
Add script for print backtrace from crashdump
without needs to downloads whole release.
Script will automatically downloads required debug symbols
from GH by tag taken from argument.

### CODE REVIEW

File: crashdump/crashdump_backtrace.sh [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+}
+
+
+# Check if all programs needed by scripts are installed
+# arg1 .. argn: programs for validate
+# On failure terminate the script and set errno
+validate_req_cmds() {
+    for cmd in $@; do
+        if [ ! $(command -v $cmd) ]; then
+            echo "Error! $cmd is not installed, please use 'sudo apt install $cmd' for install required tool"
```
Comment (SP2FET) [inline crashdump/crashdump_backtrace.sh:46]: it's worth noticing that sometimes tools have different name as a package than the executable, so package name sometimes != command name.
I'd add also more generic info like:
```
XYZ not installed, please use your package manager to install required tool. E.g. sudo apt install XYZ
```

File: crashdump/crashdump_backtrace.sh [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        if [ ! -d "$BUILD_DIR" ]; then
+            exit_abnormal "Build directory: $BUILD_DIR doesn't exists"
+        fi
+		# Check for the executable elf file
+		ELF_FILE="$BUILD_DIR/PurePhone.elf"
+        if [ ! -f "$ELF_FILE" ]; then
+            exit_abnormal "Executable file: $ELF_FILE doesn't exists"
+        fi
+		# Check for debug elf file
+		DBG_FILE="$BUILD_DIR/PurePhone.elf.debug"
```
Comment (SP2FET) [inline crashdump/crashdump_backtrace.sh:104]: so it will work only for pure?

**Review by SP2FET — APPROVED** [review]

File: crashdump/crashdump_backtrace.sh (hunk 1 above)
Comment (lucckb) [reply crashdump/crashdump_backtrace.sh:46]: fixed

File: crashdump/crashdump_backtrace.sh (hunk 2 above)
Comment (lucckb) [reply crashdump/crashdump_backtrace.sh]: Added by auto detection

**Pull request merged**